lib/sync-UpdatesModel_mongo.js

```diff
@@ -0,0 +1,41 @@
+var syncUtil = require('./sync-util');
+var MongoClient = require('mongodb').MongoClient;
+var fhdb;
+var mongo;
+
+var setFHDB = function(db) {
+  fhdb = db;
+};
+
+var setConnectionUrl = function(url) {
+  MongoClient.connect(url, function(err, db) {
+    if (err) throw err;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)